### PR TITLE
QML overlay menu

### PR DESCRIPTION
**Pull request by CODeRUS**
fixed dialogs layout
hack for dialogs mouse event (by disabling webView)

### CODE REVIEW

File: qml/OverlayButton.qml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import Qt 4.7
+import QtQuick 1.1
```
Comment (freemangordon) [inline qml/OverlayButton.qml]: Why 1.1 but not 1.0 as in the other files?

Comment (CODeRUS) [inline qml/OverlayButton.qml]: because of Image cache property, but you can remove it and use v1.0

File: qml/AddressField.qml [hunk 2]
```diff
@@ -0,0 +1,137 @@
```
Comment (romaxa) [inline qml/AddressField.qml:0]: I guess this resource missing from qmlMozEmbedTest.qrc and some others

Comment (CODeRUS) [inline qml/AddressField.qml:0]: ah, sorry, forgot about qrc.

File: qml/AddressField.qml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    height: 80 + (textInputOverlay.visible ? textInputOverlay.height : 0)
+    width: parent.width
+    
+    function focusAddressBar() {
+        addressLine.forceActiveFocus()
+        addressLine.selectAll()
+    }
+    
+    function unfocusAddressBar() {
+        addressLine.activeFocus = false
```
Comment (romaxa) [inline qml/AddressField.qml]: qrc:/qml/AddressField.qml:19: Error: Cannot assign to read-only property "activeFocus"

**Comment by romaxa** [comment]
found one issue, if you long press (UI appeared), and without release start scrolling up/down, then there is no way after that to remove that UI

**Comment by CODeRUS** [comment]
Cannot replay your bug: https://github.com/tmeshkova/qmlmozbrowser/pull/3#issuecomment-14099494

**Pull request merged**